website/docs/03-faq/index.mdx

```diff
@@ -674,23 +674,18 @@ After the boot, the first binary launched from the SD card by the firmware is th
 
 </summary>
 
-Since Onion 4.2.0, overclock is possible. The speed improvement can be very noticeable (up to 20%).
+It is possible to overclock your device starting from Onion 4.2.0. The speed improvement can be up to 20% and is very noticeable. The current overclocking procedure is a manual process.
 
-For now the overclock feature is manual.
+To overclock Retroarch for all cores, create a `cpuclock.txt` text file in the main Retroarch folder (e.g. `Retroarch/cpuclock.txt`) containing just the 4-digit Mhz number of the desired overclock speed (e.g. `1600`).
 
-Onion's overclock works with `cpuclock.txt` text files.
-Retroarch will check the presence of this file in the core config folder first, then the RA root folder.
-The content of cpuclock.txt is simply a 4-digit number(MHz).
+To overclock just a specific Retroarch core, create the `cpuclock.txt` text file within that core's configuration folder in `Saves/CurrentProfile/config/<core>/`. For example, if you want to improve the PSX speed of PCSX-ReARMed (which will allow you to enable *enhanced resolution* for many additional games) you can create a `cpuclock.txt` file containing `1600` inside `Saves/CurrentProfile/config/PCSX-ReARMed/`.
 
-The max speed overclock speed are :
+The maximum device overclock speeds are :
```

Comment:
Can we remove the spaces before all `:`? (It's a French thing)

Reply:
Had the impression that spaces before colons was the preferred style of this project because they are in other preceding sections.

Reply:
I know, but it's definitely not preferred 😅